pex/variables.py

```diff
@@ -44,6 +45,17 @@ def delete(self, variable):
   def set(self, variable, value):
     self._environ[variable] = str(value)
 
+  def _from_rc(self, rc):
+    ret_vars = {}
+    for filename in [rc, '.pexrc']:
```

Comment:
Wouldn't it make more sense to check if `.pexrc` exists *before* checking if `~/.pexrc` exists? It could be confusing if you have a `~/.pexrc` yet *mean* to use `.pexrc` that lives next to the PEX file you're trying to control. Does it make sense to consider overlaying the two environments if both exists?